src/frontend/src/modules/admin/adminViewSite/index.js

```diff
@@ -39,11 +45,27 @@ class adminViewSite extends React.Component {
     }
   }
 
+  formatDate (date) {
+    var d = new Date(date),
+        month = '' + (d.getMonth() + 1),
+        day = '' + d.getDate(),
+        year = d.getFullYear();
+
+    if (month.length < 2) 
+        month = '0' + month;
+    if (day.length < 2) 
+        day = '0' + day;
+
+    return [year, month, day].join('-');
+  }
+  
   componentDidMount () {
     var vars = {}
     window.location.href.replace(/[?&]+([^=&]+)=([^&]*)/gi, function (m, key, value) {
       vars[key] = value
     })
+    var startdate = this.formatDate('January 1, 2019')
+    var enddate = this.formatDate('January 1, 2021')
```

Comment:
Why are we hardcoding these here? If the values are passed as null through the fetch the backend gives the default values?